containerz/containerz.proto

```diff
@@ -299,3 +328,92 @@ message LogRequest {
 message LogResponse {
   string msg = 1;
 }
+
+enum Driver {
```

Comment:
Suggest to add a comment to explain what `Driver` is (since its a volume driver, but top-level description is here).